include/sys/util_internal.h

```diff
@@ -108,7 +108,6 @@
 
 #define UTIL_EVAL(...) __VA_ARGS__
 #define UTIL_EXPAND(...) __VA_ARGS__
-#define UTIL_REPEAT(...) UTIL_LISTIFY(__VA_ARGS__)
```

Comment:
Should this be deprecated instead of just being removed? I realize it is in a header named `*_internal.h`, but could it have out-of-tree users? After all, it does not have a `Z_*` prefix.

If the definition is just kept, the deprecation of `UTIL_LISTIFY` should provide a deprecated warning for this macro as well.

Reply:
ok, removed that change. It will be removed when `UTIL_LISTIFY` will be removed.

Reply:
It does seem somewhat odd to have a `internal` header file in the public `include` directory in the first place